lib/schema_generator/format.rb

```diff
@@ -270,7 +270,20 @@ def determined_definition
 
     class Links
       ALLOWED_KEYS = %w(description required minItems maxItems).freeze
-      LINKS_WITHOUT_BASE_PATHS = %w(facet_groups facets facet_values ordered_contacts ordered_foi_contacts world_locations).freeze
+      LINKS_WITHOUT_BASE_PATHS = %w(
+        facet_groups
+        facets
+        facet_values
+        ordered_contacts
+        ordered_foi_contacts world_locations
```

Comment:
```suggestion
        ordered_foi_contacts
        world_locations
```

Reply:
Thanks, surprised the linter didn't catch this!